src/nvim/testdir/Makefile

```diff
@@ -48,7 +48,7 @@ NEW_TESTS_IGNORE := $(NEW_TESTS_IN_ALOT) $(NEW_TESTS_ALOT) \
   test_listlbr \
   test_largefile \
 
-NEW_TESTS = $(addsuffix .res,$(sort $(filter-out $(NEW_TESTS_IGNORE),$(basename $(notdir $(wildcard test_*.vim))))) $(NEW_TESTS_ALOT))
+NEW_TESTS ?= $(addsuffix .res,$(sort $(filter-out $(NEW_TESTS_IGNORE),$(basename $(notdir $(wildcard test_*.vim))))) $(NEW_TESTS_ALOT))
```

Comment:
Unrelated change. @blueyed did you remove the `?=` intentionally? The top-level `Makefile` sets `NEW_TESTS` to allow running a single test like this:

    TEST_FILE=test_marks.res make oldtest

Reply:
@justinmk 
No, wasn't intentional I think.